### DISCUSSION TITLE
Exports metadata

### DISCUSSION THREAD

**Discussion by gofaster** [blakeblackshear/frigate#13789] [Ideas]
Sometimes, I move interesting exports to offline storage; where they can be viewed without access to Frigate.  The filenames are not very intuitive, apart from the camera name.  Would be be possible to add a metadata file, perhaps in JSON that lists:
```
Filename
  Name of export
  Camera
  Start timestamp (ISO 8601)
  End timestamp (ISO 8601)
  optionally Length
```

Also modify/create the metadata if the export is renamed via Frigate.

**Comment by NickM-27** [comment]
it's all in the sqlite database and you can already use the api to pull the metadata
